helics/capi.py

```diff
@@ -2819,6 +2819,8 @@ def create_message(self) -> HelicsMessage:
 class HelicsCombinationFederate(HelicsValueFederate, HelicsMessageFederate):
     pass
 
+class HelicsCallbackFederate(HelicsValueFederate, HelicsMessageFederate):
```

Comment:
This can inherit from `HelicsCombinationFederate` instead:

```suggestion
class HelicsCallbackFederate(HelicsCombinationFederate):
```